specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-01-01/imds.json

```diff
@@ -890,6 +926,136 @@
+          }
+        }
+      }
+    },
+    "PublicIpAddress": {
+      "type": "object",
+      "description": "Loadbalancer public ipAddress Metadata.",
+      "properties": {
+        "frontendIpAddress": {
+          "type": "string",
+          "description": "This is instance level public ipAddress associated to the NIC on the VM"
+        },
+        "privateIpAddress": {
+          "type": "string",
+          "description": "This is private ipAddress assigned to the NIC on the VM"
+        }
+      }
+    },
+    "LoadBalancerInboundRule": {
+      "type": "object",
+      "description": "This contains the Load Balancer inbound Rule properties.",
+      "properties": {
+        "frontendIpAddress": {
+          "type": "string",
+          "description": "This is frontend ipAddress used in load balancing rules and inbound NAT rules using which the Load Balancer directs traffic to the VM."
+        },
+        "protocol": {
+          "type": "string",
+          "description": "This is protocol in load balancing rules and inbound NAT rules using which the Load Balancer directs traffic to the VM."
+        },
+        "frontendPort": {
+          "type": "integer",
+          "description": "This is frontend port in load balancing rules and inbound NAT rules using which the Load Balancer directs traffic to the VM."
+        },
+        "backendPort": {
+          "type": "integer",
+          "description": "This is backend port using which VM communicates with Load balancer"
+        },
+        "privateIpAddress": {
+          "type": "string",
+          "description": "This is private ipAddress to the NIC using which VM communicates with Load balancer"
+        }
+      }
+    },
+    "LoadBalancerOutboundRule": {
+      "type": "object",
+      "description": "This contains the Load Balancer outbound Rule properties.",
+      "properties": {
+        "frontendIpAddress": {
+          "type": "string",
+          "description": "This is frontend IP of load balancer can be used to provide outbound connectivity to the internet for backend instances on VM."
+        },
+        "privateIpAddress": {
+          "type": "string",
+          "description": "This is private ipAddress to the NIC using which VM communicates with Load balancer"
+        }
+      }
+    },
+    "LoadBalancerTunnelInterface": {
+      "type": "object",
+      "description": "This contains the properties of tunnel interface which are configured on the VM. This is only available when VM is in the backend pool of a Gateway Load balancer. Learn more at aka.ms/gatewaylb",
+      "properties": {
+        "gatewayIpAddress": {
+          "type": "string",
+          "description": "This is private ipAddress of gateway Load Balancer of which VM is in the backend pool."
+        },
+        "gatewayPort": {
+          "type": "integer",
+          "description": "This is port for Gateway Loadbalancer to direct the traffic to VM."
```

Comment:
- "Gateway Loadbalancer" -> "Gateway Load Balancer".
- "port for" -> "port opened on the public IP of the Gateway Load Balancer". this is for making it explicit to avoid people think it is the port on the backend side, because we call the backend port the local port. so, better choose a more explicit term here.

Style matter a lot in public doc. We need to pay more attention to it.